cmp/form/DayField.js

```diff
@@ -5,29 +5,34 @@
  * Copyright © 2018 Extremely Heavy Industries Inc.
  */
 
-
+import {PropTypes as PT} from 'prop-types';
 import moment from 'moment';
 import {fmtDate} from 'hoist/format';
 import {hoistComponent, elemFactory} from 'hoist/core';
 import {dateInput} from 'hoist/kit/blueprint';
 
 import {HoistField} from './HoistField';
 
-
 /**
  * A Calendar Control for choosing a Day.
  *
- * @prop rest, see properties for HoistField
- *
- * @prop width, width of field
- * @prop popoverPosition, 'top' | 'bottom' |  'auto' (auto determined),
+ * See HoistField for properties additional to those documented below.
  */
 @hoistComponent()
 export class DayField extends HoistField {
 
-    static defaultProps = {
-        popoverPosition: 'auto'
-    }
+    static propTypes = {
+        /** Width of field */
+        width: PT.number,
```

Comment:
This is another one that seems like a lower-level thing and not specific to this particular field.